paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -587,18 +583,9 @@ class ConvMKLDNNHandlerT
       post_operations.append_sum(sum_scale);
     }
 
-    if (fuse_activation == "hard_sigmoid") {
-      post_operations.append_eltwise(activation_scale,
-                                     dnnl::algorithm::eltwise_linear,
-                                     fuse_alpha,
-                                     fuse_beta);
-      post_operations.append_eltwise(
-          activation_scale, dnnl::algorithm::eltwise_clip, 0.0f, 1.0f);
-    } else if (fuse_activation != "") {
-      const auto activation_algorithm =
-          platform::AcquireActivationAlgorithm(fuse_activation);
-      post_operations.append_eltwise(
-          activation_scale, activation_algorithm, fuse_alpha, fuse_beta);
+    if (!ctx.Attr<std::string>("fuse_activation").empty()) {
+      paddle::platform::AppendActivation(
+          ctx, post_operations, activation_scale);
```

Comment:
Yes, it's needed because conv operator is different than others. All other op+act fuse passes use default value 1.0, only `conv_mkldnn_op` computes it from attributes(`force_fp32_output`, `scale_out` and `activation_scale`).  If more ops compute `activation_scale` I will move this logic to `AppendActivation`.